pontoon/base/signals.py

```diff
@@ -17,3 +22,78 @@ def project_locale_added(sender, **kwargs):
         project = project_locale.project
         project.has_changed = True
         project.save(update_fields=['has_changed'])
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    instance = kwargs['instance']
+
+    if kwargs['raw'] or instance.translators_group:
+        return
+    try:
+        locale_ct = ContentType.objects.get(app_label='base', model='locale')
+        locale_group, _ = Group.objects.get_or_create(name='{} translators'.format(instance.code.upper()))
+        can_translate = Permission.objects.get(content_type=locale_ct, codename='can_translate_locale')
+        locale_group.permissions.add(can_translate)
+        instance.translators_group = locale_group
+    except:
```

Comment:
Never use a try block without specifying the type of error you're catching. Otherwise, any and all errors will be caught regardless of if they're expected or not.

In this case you want to be catching [ObjectDoesNotExist](https://docs.djangoproject.com/en/1.8/ref/exceptions/#objectdoesnotexist) errors.